tst/components/ui/Notification/index.test.tsx

```diff
@@ -17,13 +17,13 @@ const getNotificationComponent = () => (
 
 const getNotificationWithOptionalProps = () => (
   <Notification
-    icon={<Echo />} 
+    icon={<Echo />}
     onClose={jest.fn()}
     severity={Severity.SUCCESS}
-    message='Hello'
-    buttonProps={{ 
-      label: 'click me', 
-      onClick: jest.fn(), 
+    message="Hello"
```

Comment:
The `message` prop has the string as double quotes and the `buttonProps`'s `label` is single quote.  Checking if any reason for this?

Reply:
https://github.com/prettier/prettier/issues/656

Reply:
so prettier has taken a decision of using double quotes for jsx elements as they are more similiar to html elements and html elements use double quotes